src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/Rules/RazorGenerateWithTargetPath.cs

```diff
@@ -10,8 +10,11 @@
 
 namespace Microsoft.CodeAnalysis.Razor.ProjectSystem.Rules
 {
+    
```

Comment:
Generated file which always adds these new lines, as such would like to commit it in so we don't see it in diffs.

Reply:
Yeah I've seen this too. So annoying.